go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/longhorn/longhorn-spdk-engine
 
-go 1.21
+go 1.21.7
```

Comment:
Why do we want to use specific Go version here?

Reply:
Removed it. Thanks.